src/easier68k/core/opcodes/MOVE.py

```diff
@@ -0,0 +1,153 @@
+from ...core.enum.ea_mode import EAMode
+from ...core.enum.op_size import BigSSize
+from ...core.enum.ea_mode_bin import EAModeBinary
+from ...simulator.m68k import M68K
+import binascii
+
+# Allowed values: nothing, or some combination of B, W, and L (for byte, word, and long)
+# For example, MOVE would have 'BWL' because it can operate on any size of data, while MOVEA would have 'WL' because
+# it can't operate on byte-sized data
+allowed_sizes = 'BWL'
+
+
+class MOVE:
+    # Include parameters here
+    def __init__(self, src: EAMode, dest: EAMode, size='B'):
```

Comment:
are we certain that the default size is B?

Reply:
I'm not, I'll check that.

Reply:
After a test, can confirm that default size is W. Not sure if this applies to all commands or just move, we'll need to investigate.